test_conformance/conversions/basic_test_conversions.h

```diff
@@ -268,8 +282,8 @@ struct IterOverTypes : public TestType
         if (!testType<OutType>(outType)) vlog_error("Unexpected data type!\n");
 
```

Comment:
We need some extra checks now, to differentiate between a `cl_half` and a `cl_ushort`.  I'm not entirely sure if this is the right fix but this seems to be helping me to get farther.

```suggestion
        if (In != inType) return;
        if (Out != outType) return;

```

Reply:
Neither of these conditions is ever met during test at my machine. In general those values shouldn't differ, the main difference is that `In`/`Out` are template arguments known during build time. 

To distinguish between `cl_half` and `cl_ushort` `isTypeFp` array was introduced and it is used on line below as an template argument.

Reply:
This doesn't seem to be needed with the latest changes.  Resolving.